hknweb/templates/registration/login.html

```diff
@@ -1,34 +1,148 @@
 {% extends "base.html" %}
 {% load static %}
+{% block header %}
+<style>
```

Comment:
I think it would be better to have these in a style.css file

Reply:
Resolved